scripts/main-run

```diff
@@ -55,16 +55,21 @@ $FIREWALLA_HOME/scripts/main-start
 sudo rm -rf /media/root-rw/overlay.bak
 sudo rm -rf /media/root-rw/overlay-workdir.bak
 
-# this will touch and init watchdog
-$FIREWALLA_HOME/scripts/fire-ping.sh
-sleep 1
-$FIREWALLA_HOME/scripts/fire-ping.sh
-echo "Launching WatchDog"
-sleep 1
-$FIREWALLA_HOME/scripts/fire-ping.sh
-sudo service watchdog stop
-echo "WatchDog Stopped"
-$FIREWALLA_HOME/scripts/fire-ping.sh
-sleep 3 
-sudo service watchdog start
-echo "WatchDog Started"
+if [ -e /home/pi/.firewalla/config/disable_watchdog ]; then
+    logger "FIREWALLA:WATCHDOG:DISABLED"
```

Comment:
should call "sudo systemctl stop watchdog" when the disable flag exists, in case it's already started.

Reply:
added